doc/userguide/performance/hyperscan.rst

```diff
@@ -4,133 +4,29 @@ Hyperscan
 Introduction
 ~~~~~~~~~~~~
 
-"Hyperscan is a high-performance multiple regex matching library." https://www.hyperscan.io
+"Hyperscan is a high performance regular expression matching library..." https://www.intel.com/content/www/us/en/developer/articles/technical/introduction-to-hyperscan.html
```

Comment:
Suggestion:
```diff
- "Hyperscan is a high performance regular expression matching library..." https://www.intel.com/content/www/us/en/developer/articles/technical/introduction-to-hyperscan.html
+ "Hyperscan is a high performance regular expression matching library (...)" (cf. 
+ https://www.intel.com/content/www/us/en/developer/articles/technical/introduction-to-hyperscan.html)
```

Reply:
That was my thinking as well, do we want to do the advanced installation instructions as part of this PR?

Reply:
If that's ok with you, this is good, as we ensure that we won't remove them and not bring them back.
But we can also just make sure that these two changes go into the same release, IMHO, and then they don't have to be part of the same PR.